app/templates/components/public/ticket-list.hbs

```diff
@@ -88,10 +88,10 @@
                     @id={{concat ticket.id "_quantity"}}
                     @value={{ticket.orderQuantity}} />
                   <i class="dropdown icon"></i>
-                  <div class="default text">0</div>
+                  <div class="default text">{{if (eq this.tickets.length 1) (max ticket.minOrder 1) 0}}</div>
```

Comment:
Wouldn't `{{ticker.orderQuantity}}` work here?

Reply:
no bcz the value of ticket.orderQuantity is undefined intially and meantime the dom is rendered completely after that ticket.orderQuantity value is changed.

Reply:
but I think it should be working but its not. it is showing blank

Reply:
`{{or ticket.orderQuantity 0}}`

Reply:
this will work OK for more then one type of tickets but in case of only 1 type of tickets still ticket.orderQuanitity is undefined so it is 0 there also.
and I don't know why in this it is wokring correct
```handlebars
 <Input
    @type="hidden"
    @id={{concat ticket.id "_quantity"}}
    @value={{ticket.orderQuantity}} />
```
but not here
```handlebars
 <div class="default text">{{or ticket.orderQuanitity 0}}</div>
```